contracts/staking/contracts/src/staking_pools/MixinStakingPool.sol

```diff
@@ -76,6 +76,11 @@ contract MixinStakingPool is
         external
         returns (bytes32 poolId)
     {
+        // Operator address is used to check whether pool has been created or not
```

Comment:
On second thought, this isn't necessary. No one has access to the private key for address zero, so this check is redundant. I think that I had an inaccurate picture of the flow for creating a staking pool previously, which is why I left the original comment. For some reason, I was under the impression that a pool could be created where the `sender` was not the same as the `operator`. 

Given more context, I'd say that we should remove this check. Sorry for flip-flopping on you here.

Reply:
Agreed!